Fabric/src/main/java/me/ferdz/placeableitems/PlaceableItems.java

```diff
@@ -1,20 +1,29 @@
 package me.ferdz.placeableitems;
 
+import me.ferdz.placeableitems.client.PlaceableItemsClientHandler;
 import me.ferdz.placeableitems.init.PlaceableItemBlockEntityRegistry;
 import me.ferdz.placeableitems.init.PlaceableItemsBlockRegistry;
 import me.ferdz.placeableitems.init.PlaceableItemsItemsRegistry;
+
 import net.fabricmc.api.ClientModInitializer;
 import net.fabricmc.api.ModInitializer;
+import net.fabricmc.fabric.api.network.ServerSidePacketRegistry;
+
+import net.minecraft.util.Identifier;
 
 import java.util.logging.Logger;
 
 public class PlaceableItems implements ModInitializer, ClientModInitializer {
 
     public static final String MODID = "placeableitems";
 
+    public static final Identifier PACKET_ID_NOTIFY_ITEM_PLACE_KEY = new Identifier(PlaceableItems.MODID, "notify_item_place_key");
+
     // Directly reference a log4j logger.
     public static Logger logger = Logger.getLogger("placeableitems");
 
+    private static boolean keyPressed = false;
+
```

Comment:
This probably doesn't belong in the mod class itself, it should be moved to something dedicated for the key presses / input handling

Reply:
Unfortunately I had no other place to put it. Having it be in the client class would cause class loader issues when accessed from the server and mixins will not load if its class contains static methods. Having this value be in its own class seems a bit awkward. I could still move it if you think it’s better in terms of organization but I figured I’d give my reasoning for putting it here instead